packages/consumption/src/modules/attributes/local/LocalAttribute.ts

```diff
@@ -8,10 +8,11 @@ import {
     RelationshipAttribute,
     RelationshipAttributeJSON
 } from "@nmshd/content";
-import { CoreAddress, CoreDate, CoreId, CoreSynchronizable, ICoreDate, ICoreId, ICoreSynchronizable } from "@nmshd/transport";
+import { CoreAddress, CoreDate, CoreId, ICoreDate, ICoreId } from "@nmshd/core-types";
+import { CoreSynchronizable, ICoreSynchronizable } from "@nmshd/transport";
```

Comment:
Can CoreSynchronizable (and CoreSerializable) also move to core-types?

Reply:
CoreSerializable is a utility type that would bring crypto to the core-types library which we dont want. The long-running plan from my side is to remove this type completely. This will introduce more changes so I will do this in a next iteration.

CoreSynchronizable is a data type strictly coupeled to how the transport sync works so I have not moved this.

Reply:
To not have more confusion I remove CoreSerializable completely.